sycl/test/basic_tests/accessor/accessor_property_list_rt.cpp

```diff
@@ -10,16 +10,16 @@ using namespace sycl::ONEAPI;
 int main() {
   {
     // Single RT property
-    accessor_property_list PL{sycl::noinit};
+    accessor_property_list PL{sycl::no_init};
     static_assert(!PL.has_property<property::no_offset>(), "Property is found");
-    assert(PL.has_property<sycl::property::noinit>() && "Property not found");
+    assert(PL.has_property<sycl::property::no_init>() && "Property not found");
```

Comment:
Probably it make sense to test both spellings: SYCL 2020 and ONEAPI extension (i.e. deprecated).

Reply:
Done.